server/form-pages/apply/add-documents/attachDocuments.test.ts

```diff
@@ -124,6 +124,15 @@ describe('attachDocuments', () => {
         [`selectedDocuments_${selectedDocuments[1].id}`]: `You must enter a description for the document '${selectedDocuments[1].fileName}'`,
       })
     })
+    it('should return an error if the user has selected more than 5 documents', () => {
+      const selectedDocuments = Array(6)
+        .fill(null)
+        .map((_, idx) => documentFactory.build({ fileName: `file${idx}.pdf`, description: 'document description' }))
```

Comment:
The factories have a `buildList` method to build... lists 😄  So you should be able to create the array of docs like this:

```suggestion
      const selectedDocuments = documentFactory.buildList(6)
```

This will return an array, so the rest of the test should be able to remain unchanged.

Reply:
Nice!